docs/cli/pushing-data-to-pulse.md

```diff
@@ -24,15 +24,7 @@ Consider the following before setting up the integration using the Pulse CLI:
 
 -   [**Make sure that you have an API key**](https://app.pulse.codacy.com/integrations/cli){: target="_blank"} provided by Pulse to identify your organization and authorize you to send data to Pulse.
 
--   In some scenarios, it may not be feasible to use the CLI to send data to Pulse, such as when reporting changes or incidents, or if you are reporting events from providers that only support webhooks.
-
-    For these situations, you can call an HTTP POST webhook instead. For example:
-
-    ```text
-    https://ingestion.pulse.codacy.com/v1/ingestion/<PROVIDER>?api_key=<API KEY>
-    ```
-
-    **Please let us know if you're planning on reporting events to Pulse using the webhook** so we can give you more detailed instructions on how to use it.
+-   In some scenarios, it may not be feasible to use the CLI to send data to Pulse, such as when reporting changes or incidents, or if you are reporting events from providers that only support webhooks. For these situations, you can [use the Pulse Ingestion API instead](https://ingestion.pulse.codacy.com/v1/api-docs){: target="_blank"}.
```

Comment:
```suggestion
-   In some scenarios, it may not be feasible to use the CLI to send data to Pulse, such as when you have the data inside an application. For these situations, you can [use the Pulse Ingestion API instead](https://ingestion.pulse.codacy.com/v1/api-docs){: target="_blank"}.
```

I think the webhooks part is not worth mentioning since it is provided automatically inside the integrations.
And you need to get the proper url from our app and cannot do the setup without it.

Reply:
You're right, and I also like that the explanation becomes simpler. :+1: